src/ios/CDVCamera.m

```diff
@@ -240,13 +240,9 @@ - (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)butto
 
 - (void)repositionPopover:(CDVInvokedUrlCommand*)command
 {
-    if (([[self pickerController] pickerPopoverController] != nil) && [[[self pickerController] pickerPopoverController] isPopoverVisible]) {
+    NSDictionary* options = [command argumentAtIndex:0 withDefault:nil];
```

Comment:
Hi.
Thanks for the PR.

I think you forked you repo before this changes were made and didn't sync before sending your PR, but this code is important to fix another bug.
Can you make sure that your fork is in sync with the latest code on this repo and then send the PR with just the relevant code to fix the issue?